pilco/models/smgpr.py

```diff
@@ -4,51 +4,53 @@
 
 from .mgpr import MGPR
 
-float_type = gpflow.settings.dtypes.float_type
+from gpflow import config
+float_type = config.default_float()
 
 
 class SMGPR(MGPR):
-    def __init__(self, X, Y, num_induced_points, name=None):
-        gpflow.Parameterized.__init__(self, name)
+    def __init__(self, data, num_induced_points, name=None):
         self.num_induced_points = num_induced_points
-        MGPR.__init__(self, X, Y, name)
+        MGPR.__init__(self, data, name)
 
-    def create_models(self, X, Y):
+    def create_models(self, data):
         self.models = []
         for i in range(self.num_outputs):
-            kern = gpflow.kernels.RBF(input_dim=X.shape[1], ARD=True)
+            kern = gpflow.kernels.SquaredExponential(lengthscales=tf.ones([data[0].shape[1],], dtype=float_type))
```

Comment:
Is `ARD=True` by default now?

Reply:
ARD is not a keyword argument, but by initialising the lengthscales to a specific shape, the kernel implements ARD. I found that a bit weird too.